official-antistasi-community/A3-Antistasi
# Setup GUI Factions Info

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [X] Enhancement

### What have you changed and why?
Information:
Added more information and lore blurbs in the Setup UI. Also merges the addon and DLC into one panel.
    
### Please specify which Issue this PR Resolves.
closes #3221 

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Needs some more testing to ensure EVERYTHING displays properly
Currently, the addon menu acts as a different box and doesnt let you click the switch button, and the addon menu disappears until updated when the selection is changed, but given how few people actually enable addon packs I'm going to fix it in a later commit.
********************************************************
Notes:
